File: Emlak/src/app/admin/model/IFormElemanGrup.ts
import { IFormEleman } from './IFormEleman';

export interface IFormElemanGrup {
    ID: number,
    //[Required(ErrorMessage = "Baþlýk alaný boþ olamaz ve en fazla 50 karakter olmalýdýr.")]
    //[StringLength(50)]
    Title: string,
    Description: string,
    Code: string,
    Active: boolean,

    Mesaj: string,

    PropertyList: Array<IFormEleman>
}

File: AdminPanel/Repository/FormElemanGrup/IFormElemanGrup.cs
using Repository.FormElemanModel;
using System.Collections.Generic;

namespace Repository.FormElemanGrupModel
{
    public interface IFormElemanGrup
    {
        int ID { get; set; }
        string Title { get; set; }
        string Description { get; set; }
        string Code { get; set; }
        bool Active { get; set; }

        string Mesaj { get; set; }

        List<FormEleman> PropertyList { get; set; }

        List<FormElemanGrup> List();
        IFormElemanGrup Select(int id);
        bool Insert(IFormElemanGrup formeleman);
        IFormElemanGrup Update(int id, IFormElemanGrup formeleman);
        bool Update(IFormElemanGrup formeleman);
        bool Delete(int id);
        bool Copy(int id);
    }
}
